hy/cmdline.py

```diff
@@ -363,8 +376,8 @@ def cmdline_handler(scriptname, argv):
     return run_repl(spy=options.spy, output_fn=options.repl_output_fn)
 
 
-# entry point for cmd line script "hy"
 def hy_main():
+    """Entry point for cmd line script `hy`"""
```

Comment:
All these transforms from comments to docstrings are good, but should also get their own commit. (A lot of important stuff is going on in this commit that we don't want crowded out by minor stuff like this.)